cg/models/cg_config.py

```diff
@@ -85,6 +85,12 @@ class DemultiplexConfig(BaseModel):
     slurm: SlurmConfig
 
 
+class DownsampleConfig(BaseModel):
+    downsample_dir: str
+    downsample_script: str
```

Comment:
Do we need to prepend "downsample" given the class name?

Reply:
I just took over how it was previously and will leave it at that. We can rework at another time if it is wished.